src/app/shared/components/reject-modal-window/reject-modal-window.component.ts

```diff
@@ -0,0 +1,42 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { Constants } from 'src/app/shared/constants/constants';
+import { Application } from 'src/app/shared/models/application.model';
+import {
+  FormBuilder,
+  FormControl,
+  FormGroup,
+  Validators,
+} from '@angular/forms';
+import { MatDialogRef } from '@angular/material/dialog';
+
+@Component({
+  selector: 'app-reject-modal-window',
+  templateUrl: './reject-modal-window.component.html',
+  styleUrls: ['./reject-modal-window.component.scss'],
+})
+export class RejectModalWindowComponent implements OnInit {
+  readonly constants: typeof Constants = Constants;
+
+  modalTitle = 'ВІДМОВИТИ';
+  modalDescription =
+    'Ви впевнені, що хочете перевести заяву у статус ”Відмовлено”?';
+
+  ReasonFormGroup: FormGroup;
+
+  @Input() application: Application;
```

Comment:
usually, @Input and @Output go first, after this is a declaration of properties. Check the code style angular guideline. We are not very strict with that, but Input/Output should go top

Reply:
updated this